lib/segment/src/segment.rs

```diff
@@ -211,11 +211,17 @@ impl Segment {
             match vector_opt {
                 None => {
                     let dim = vector_storage.vector_dim();
-                    let vector: Vector = match *vector_storage {
+                    // placeholder vector for marking deletion
+                    let vector = match *vector_storage {
                         VectorStorageEnum::DenseSimple(_)
                         | VectorStorageEnum::DenseMemmap(_)
-                        | VectorStorageEnum::DenseAppendableMemmap(_) => vec![1.0; dim].into(),
-                        VectorStorageEnum::SparseSimple(_) => SparseVector::default().into(),
+                        | VectorStorageEnum::DenseAppendableMemmap(_) => {
+                            Vector::from(vec![1.0; dim])
+                        }
+                        VectorStorageEnum::SparseSimple(_) => Vector::from(SparseVector::default()),
+                        VectorStorageEnum::MultiDenseSimple(_) => {
+                            Vector::from(vec![vec![1.0; dim]])
```

Comment:
I don't know if empty multivectror is the correct case. If yes, I propose to use an empty vector

Reply:
If seems multivector cannot be empty because score is not defined

Reply:
What do you think about my choice?

Reply:
There are no definition of empty multi vector score, empty multivector is invalid. It means your choice is excellent